src/ocsp.c

```diff
@@ -1064,6 +1064,31 @@ int wolfSSL_i2d_OCSP_CERTID(WOLFSSL_OCSP_CERTID* id, unsigned char** data)
     return id->rawCertIdSize;
 }
 
+WOLFSSL_OCSP_CERTID* wolfSSL_d2i_OCSP_CERTID(WOLFSSL_OCSP_CERTID** cidOut,
+                                             const unsigned char** derIn,
+                                             int length)
+{
+    if ((cidOut == NULL) || (derIn == NULL) || (length == 0))
+        return (NULL);
```

Comment:
Remove parentheses around return value here and elsewhere for other return values (e.g. line 1079).